docs/develop/clients/ios.md

```diff
@@ -1,3 +1,129 @@
-Documentation for Textile's iOS SDK (ios-textile) can be found at https://github.com/textileio/ios-textile.
+The iOS SDK is designed to work in iOS mobile apps written in Objective-C or Swift. 
 
-CocoaPods pod: https://cocoapods.org/pods/Textile
+Apps running the iOS SDK run an on device Textile peer (including a managed [IPFS](https://ipfs.io) peer). The SDK provides you with a number of helpful APIs to manage sharing, contacts, file encryption & storage, and much more.
+
+Below are some basic examples to get you started. If you are interested in a more thorough walk-through, check out the [Tour of Textile's](/a-tour-of-textile) examples using Swift or Objective-C or jump to the full API documentation if you are ready.
+
+## Installation
+
+The iOS is published as a [pod on CocoaPod.org](https://cocoapods.org/pods/Textile). The library is also [published on GitHub](https://github.com/textileio/ios-textile).
+
+#### CocoaPods
+
+If you haven't already, you'll need to [setup CocoaPods in your project](https://guides.cocoapods.org/using/using-cocoapods.html). Next, add Textile to your project.
+
+
+```cmd
+pod 'Textile'
+```
+
+This will install the latest version of the SDK into your app. 
+
+#### Carthage
+
+```cmd
+github "textileio/ios-textile"
```

Comment:
Would be amazed if this works. Does it?

Reply:
it works! not certain about dependencies though. might just make the explicit to be safe

Reply:
coooool. maybe add some "experimental feature" blurb under it.